docs/xrunjdwp.md

```diff
@@ -0,0 +1,118 @@
+﻿<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -Xrunjdwp
+
+The `-Xrunjdwp` option enables the Java&trade; Debug Wire Protocol (JDWP) agent in the VM. The JDWP agent provides a connection point for the debugger to debug a Java application in that VM.
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Notes:**
+
+- You can enable the JDWP support on Checkpoint/Restore In Userspace (CRIU) restore by specifying the following command-line options pre-checkpoint:
+
+    - `-Xrunjdwp`
+    - `-agentlib:jdwp`
+    - `-agentpath:/path/to/libjdwp.so`
+
+- The main difference between `-Xrunjdwp` and `-agentlib:jdwp` is that `-Xrunjdwp` runs the JDWP agent as a separate process, while `-agentlib:jdwp` loads the JDWP agent as a Java library. That means that you can use `-Xrunjdwp` to debug a Java application that is running in a separate VM, and `-agentlib:jdwp` to debug a Java application that is running in the same VM. For more information, see [VM Invocation Options](https://docs.oracle.com/en/java/javase/22/docs/specs/jpda/conninv.html#vm-invocation-options).
```

Comment:
The difference between `-Xrunjdwp` and `-agentlib:jdwp` was for historical reasons, I think OpenJ9 loads the same `jdwp` agent library eventually. 
Lines 36 & 37 can be removed.